test/unit/storage_sync_test.rb

```diff
@@ -115,9 +107,8 @@ def test_sentinels_simple_url
     }
 
     conn = StorageSync.send :new, Storage::Helpers.config_with(config_obj)
-    assert_sentinel_connector(conn)
-    assert_client_config(conn, url: "redis://#{config_obj[:url]}",
-                               sentinels: [{ host: '127.0.0.1', port: 26_379 }])
+    assert_sentinel_config(conn, url: "redis://#{config_obj[:url]}",
+                           sentinels: [{ host: '127.0.0.1', port: 26_379 }])
```

Comment:
I hope we at least log any `a_malformed_url` detected. This is to inform users about typos.